src/Service.Tests/SqlTests/GraphQLMutationTests/DwSqlGraphQLMutationTests.cs

```diff
@@ -434,6 +434,154 @@ public override async Task DeleteMutationWithOnlyTypename()
+           SELECT 1
+           FROM [books] AS [table0]
+           WHERE [table0].[title] = 'Random Book'
+             AND [table0].[publisher_id] = 1234
+         ) THEN 'true' 
+         ELSE 'false'
+       END AS BookExists
+            ";
+
+            string graphQLMutationName = "executeInsertBook";
+            string graphQLMutation = @"
+                mutation {
+                    executeInsertBook(book_id: 10, title: ""Random Book"", publisher_id: 1234 ) {
+                        result
+                    }
+                }
+            ";
+
+            string currentDbResponse = await GetDatabaseResultAsync(msSqlQuery);
+            Assert.AreEqual(currentDbResponse, "False", "Entry should not exist in the db before insertion");
+            JsonElement graphQLResponse = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+
+            // Stored Procedure didn't return anything
+            SqlTestHelper.PerformTestEqualJsonStrings("[]", graphQLResponse.ToString());
+
+            // check to verify new element is inserted
+            string updatedDbResponse = await GetDatabaseResultAsync(msSqlQuery);
+            Assert.AreEqual(updatedDbResponse, "True", "Entry should be inserted into db.");
+        }
+
+        /// <summary>
+        /// <code>Do: </code>Book title updation and return the updated row
+        /// <code>Check: </code>if the result returned from the mutation is correct and
+        /// updated value exists in the database
+        /// </summary>
+        [TestMethod]
+        public async Task TestStoredProcedureMutationForUpdate()
+        {
+            string dbQuery = @"
+SELECT CASE
+         WHEN EXISTS (
+           SELECT 1
+           FROM [books] AS [table0]
+           WHERE [table0].[id] = 14
+             AND [table0].[publisher_id] = 1234
+             AND [table0].[title] = 'Before Midnight'
+         ) THEN 'true' ELSE 'false'
+       END AS BookExists
+            ";
+
+            string graphQLMutationName = "executeUpdateBookTitle";
+            string graphQLMutation = @"
+                mutation {
+                    executeUpdateBookTitle(id: 14, title: ""Before Midnight"") {
+                        id
+                        title
+                    }
+                }
+            ";
+
+            JsonElement graphQLResponse = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+            string updatedValueExists = await GetDatabaseResultAsync(dbQuery);
+            string insertedValue = "{\"id\":14,\"title\":\"Before Midnight\"}";
+            Assert.IsTrue(updatedValueExists.Equals("True"), "Title should be updated for given book id.");
+            Assert.IsTrue(graphQLResponse.ToString().Contains(insertedValue), "GraphQL response should contain the updated entry");
+        }
+
+        /// <summary>
+        /// <code>Do: </code>deletes a Book and return nothing
```

Comment:
nit: not "nothing" , returns `result` field with metadata about successful execution/records affected?

Reply:
fixed.